src/__mocks__/ppl-test-data/singleLineQueries.ts

```diff
@@ -479,3 +479,372 @@ export const queryWithFunctionCalls = {
     ],
   ] as monacoTypes.Token[][],
 };
+
+export const joinQuery = {
+  query:
+    'left outer join left = table1 right = table2 left_hint.id = leftIdent on isnull(leftIdent) = false right = table2',
```

Comment:
is `left` supposed to be suggested? when i type `left` i don't get that as a suggestion, it also isn't followed by `outer` as a suggestion.

Reply:
I left that out of the suggestions cause I thought it could be confusing bundling it with the commands. But it makes sense to suggest outer and join after left/right, will add that